package.json

```diff
@@ -197,7 +197,6 @@
 		"sass-loader": "^10.2.0",
 		"semver": "^7.3.2",
 		"shelljs": "^0.8.5",
-		"storybook-vue-router": "^1.0.7",
```

Comment:
This project is basically abandoned. It doesnt officially support storybook > v5 and is requiring `--legacy-peer-deps` to install correctly